exercises/04_power/power.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func pow(a int, b int) int {
+	if b == 0 {
+		return 1
+	}
+	res := a
+	for i := b; i > 1; i, res = i-1, res*a {
```

Comment:
Separating looping logic and looped logic is a good style. Usually this means that there should be something in the for body. See the comment above for an example.